<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyBoard</class>
 <widget class="QWidget" name="KeyBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="r_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="v_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delet_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>38</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="w_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>w</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finish_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>finish</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="d_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>d</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QPushButton" name="j_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>j</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upper_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>38</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>A/a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="a_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="k_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="e_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clr_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="o_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>l</string>
    </property>
   </widget>
   <widget class="QPushButton" name="g_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QPushButton" name="u_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>u</string>
    </property>
   </widget>
   <widget class="QPushButton" name="h_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="i_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0099cc;
color:white;
border-radius:5;
</string>
    </property>
    <property name="text">
     <string>q</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
